projects/plugins/jetpack/modules/comment-likes.php

```diff
@@ -9,6 +9,8 @@
  * Auto Activate: No
  * Module Tags: Social
  * Additional Search Queries: like widget, like button, like, likes
+ *
+ * @package Jetpack
```

Comment:
```suggestion
 * @package automattic/jetpack
```

Moving to this format.

Reply:
Nice catch! Should be updated now.